splink/linker.py

```diff
@@ -173,6 +175,11 @@ def two_dataset_link_only(self):
         else:
             return False
 
+    def _create_schema(self, table_name):
```

Comment:
Could we call this `_prepend_schema_to_table_name` for clarity

Reply:
Completely agree.